src/Configuration/NodeDefinition/SshNode.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Configuration\NodeDefinition;
+
+use Symfony\Component\Config\Definition\Builder\NodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+
+class SshNode implements NodeDefinitionInterface
+{
+    public static function create(): NodeDefinition
+    {
+        $builder = new TreeBuilder();
+        $node = $builder->root('ssh');
+
+        // @formatter:off
+        $node
+            ->children()
+                ->booleanNode('enabled')->end()
+                ->arrayNode('keys')
+                    ->children()
+                        ->scalarNode('private')->end()
```

Comment:
Tohle může pryč https://github.com/keboola/internal/issues/307#issuecomment-526091066

Reply:
je to pryč